### PR TITLE
 kubernetes-helm/monocular#412 Add Values in API + Update support 

### PR DESCRIPTION
**Pull request by sebadiaz**
kubernetes-helm/monocular#412 Add Values in API

### CODE REVIEW

**Comment by sebadiaz** [comment]
I have Rewrited code in new repo

**Comment by sebadiaz** [comment]
@prydonius done

**Comment by prydonius** [comment]
thanks @sebadiaz, will give this a run soon, sorry for the delay

**Comment by sebadiaz** [comment]
!

File: src/api/Makefile [hunk 1]
```diff
@@ -23,11 +23,11 @@ all:
 	@echo "Use a Makefile to control top-level building of the project."
```
Comment (prydonius) [inline src/api/Makefile]: can you revert these changes?

Comment (sebadiaz) [reply src/api/Makefile]: Done

**Comment by jroucheton** [comment]
It will be awesome!

**Comment by sebadiaz** [comment]
Muchas Gracias !
 :satisfied:

**Comment by prydonius** [comment]
As we start working on Monocular 1.0 (read the proposal: https://github.com/helm/community/blob/master/proposals/monocular-1.0-improvements.md), we will be removing in-cluster features from Monocular to keep the project focused on the public search and discovery site experience (e.g. hub.kubeapps.com and hub.helm.sh in the future). For in-cluster features, I recommend you look into [Kubeapps](https://github.com/kubeapps/kubeapps) or possible the [RedHat Automation Broker](https://blog.openshift.com/automation-broker-discovering-helm-charts/).

Thanks a lot for the contribution!
